geopatterns_demo/models.py

```diff
@@ -35,3 +35,10 @@ class PhraseDescription(BaseModel):
     """
 
     text: str
+
+
+class Thesaurus(Enum):
+    """List of dictionaries with words."""
+
+    ADJECTIVES = 'geopatterns_demo/dictionary/adjectives.txt'
```

Comment:
```suggestion
    ADJECTIVES = Path(__file__).parent / 'dictionary/adjectives.txt'
```

Check out `Path` class from `pathlib` library. It helps working on file paths A LOT.

Reply:
Whether there is a rule for the correct order of classes. Let's say alphabetically?